src/api/keystore.coffee

```diff
@@ -168,7 +188,7 @@ exports.getCertKeyStatus = getCertKeyStatus = (callback) ->
   Keystore.findOne (err, keystoreDoc) ->
     return callback err, null if err
 
-    pem.getModulus keystoreDoc.key, (err, keyModulus) ->
+    pem.getModulusFromProtected keystoreDoc.key, keystoreDoc.passphrase, (err, keyModulus) ->
```

Comment:
What happens if you call this with a key that isn't protected, will it still work?